ache/src/main/java/achecrawler/rest/RestServer.java

```diff
@@ -124,6 +124,8 @@ public void start() {
         server.post("/crawls/{crawler_id}/labels",     labelsResource.addLabels);
         server.get( "/crawls/{crawler_id}/_search",    elasticsearchProxyResource.searchApi);
         server.post("/crawls/{crawler_id}/_search",    elasticsearchProxyResource.searchApi);
```

Comment:
Now that `ElasticsearchProxyResource.searchApi()` forwards requests to `_msearch`, this endpoint `_search` will probably not work and is not required anymore. So it might be better to completely remove lines 125-126.